tests/scalers_go/azure_queue_pause/azure_queue_pause_test.go

```diff
@@ -0,0 +1,248 @@
+//go:build e2e
```

Comment:
This e2e test doesn't check azure queue itself but scaling internals (the pause annotation), 
Could you move to `internals` folder and remove the reference to the scaler (which isn't important in what is tested)?